src/styles/demo.scss

```diff
@@ -0,0 +1,170 @@
+@import "_main";
+
+html,
+body {
+    @include flex();
+    @include flex-wrap();
+    position: relative;
+    width: 100%;
+    height: 100%;
+    
+    @include body();
+    font-family: 'Roboto', sans-serif;
+    font-size: 16px;
+}
+.header,
+.main,
+.footer {
```

Comment:
Please don't use class names that have the same name of the tag. 
Also a line space as above